_posts/2017-04-02-generating-sitemaps-with-apache-spark.markdown

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title: "Generating Sitemaps with Apache Spark"
+date: 2017-04-02
+comments: true
+author: db
+categories: [Big Data, Apache Spark]
+---
+While Artsy is the largest database of Contemporary Art online, it's not exactly "big data". To date, we have published 604,897 artworks by 69,110 artists from 4,192 partners (galleries, museums, institutions) in 43,749 shows. Our team has written 8,778 articles, hosted 187 art fairs and a few dozen auctions. We have 1,015 genes from the Art Genome project.
```

Comment:
Yeah, I'll use the media stats numbers although they are close and anyone who has a computer can get our exact public counts ;)